File: 백준/Bronze/10833. 사과/사과.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        int left = 0;
        for (int i = 0; i < n; i++) {
            int studentN = sc.nextInt();
            int appleN = sc.nextInt();

            left += appleN % studentN;
        }

        System.out.println(left);
    }
}

File: 백준/Bronze/10833. 사과/사과.cc
#include <iostream>
using namespace std;

int main() {
	int n;
	cin >> n;

	int left = 0;
	for (int i = 0; i < n; i++)
	{
		int studentN;
		int appleN;

		cin >> studentN >> appleN;

		left += appleN % studentN;
	}

	cout << left << endl;
}